### PR TITLE
Expands Slapping: Facepalms and slapping people awake

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

- Slapping yourself while targeting the face will now deliver a facepalm.
- Slapping someone while targeting the face who's asleep or unconscious will wake them up slightly. It's worse than just using help intent but it's funnier. 

## Why It's Good For The Game

There's no facepalm emote for carbons. Wack. Allows humans to facepalm with a funny sound. 
Slapping people to wake them up is a classic time-honored tradition.

## Changelog

:cl: Melbert
expansion: Slapping yourself in the face now delivers a facepalm.
expansion: Slapping someone who's asleep in the face may wake them up faster.
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
give yourself code improvement for ultimate gbp farming

**Comment by MrMelbert** [comment]
> gbp farming

image

**Comment by SuperNovaa41** [comment]
no gbp will ever be enough

File: code/game/objects/items/hand_items.dm [hunk 1]
```diff
@@ -213,35 +213,60 @@
...
 	else if(user.zone_selected == BODY_ZONE_HEAD || user.zone_selected == BODY_ZONE_PRECISE_MOUTH)
-		user.visible_message(span_danger("[user] slaps [M] in the face!"),
-			span_notice("You slap [M] in the face!"),
-			span_hear("You hear a slap."))
+		if(user == slapped)
+			user.visible_message(
+				span_danger("[user] facepalms!"),
+				span_notice("You facepalm."),
+				span_hear("You hear a slap."),
+				)
```
Comment (tralezab) [inline code/game/objects/items/hand_items.dm:244]: do this for all of your visible messages, unindent the last )

**Review by tralezab — CHANGES_REQUESTED** [review]
your indentation reps, melb!

**Pull request merged**